osu.Game/Screens/Select/BeatmapInfoWedgeV2.cs

```diff
@@ -0,0 +1,341 @@
+
+            private ModSettingChangeTracker? settingChangeTracker;
+
+            private IBindable<StarDifficulty?>? starDifficulty;
+            private CancellationTokenSource? cancellationSource;
+
+            public WedgeInfoText(WorkingBeatmap working)
+            {
+                this.working = working;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(LocalisationManager localisation)
+            {
+                var beatmapInfo = working.BeatmapInfo;
+                var metadata = working.Metadata;
+
+                RelativeSizeAxes = Axes.Both;
+
+                titleBinding = localisation.GetLocalisedBindableString(new RomanisableString(metadata.TitleUnicode, metadata.Title));
+                artistBinding = localisation.GetLocalisedBindableString(new RomanisableString(metadata.ArtistUnicode, metadata.Artist));
+
+                Children = new Drawable[]
+                {
+                    new FillFlowContainer
+                    {
+                        Name = "Topright-aligned metadata",
+                        Anchor = Anchor.TopRight,
+                        Origin = Anchor.TopRight,
+                        Direction = FillDirection.Vertical,
+                        Padding = new MarginPadding { Top = 3, Right = 8 },
+                        AutoSizeAxes = Axes.Both,
+                        Shear = wedged_container_shear,
+                        Spacing = new Vector2(0f, 5f),
+                        Children = new Drawable[]
+                        {
+                            starRatingDisplay = new StarRatingDisplay(default, animated: true)
+                            {
+                                Anchor = Anchor.TopRight,
+                                Origin = Anchor.TopRight,
+                                Shear = -wedged_container_shear,
+                                Alpha = 0f,
+                            },
+                            new BeatmapSetOnlineStatusPill
+                            {
+                                AutoSizeAxes = Axes.Both,
+                                Anchor = Anchor.TopRight,
+                                Origin = Anchor.TopRight,
+                                Shear = -wedged_container_shear,
+                                TextSize = 11,
+                                TextPadding = new MarginPadding { Horizontal = 8, Vertical = 2 },
+                                Status = beatmapInfo.Status,
+                                Alpha = string.IsNullOrEmpty(beatmapInfo.DifficultyName) ? 0 : 1
+                            }
+                        }
+                    },
+                    new FillFlowContainer
+                    {
+                        Name = "Top-left aligned metadata",
+                        Direction = FillDirection.Vertical,
+                        Position = new Vector2(text_margin + shear_width, 12),
+                        Width = .7f,
```

Comment:
This is quite losing on the extra space:
![image](https://user-images.githubusercontent.com/35318437/212605014-78e1a8e2-e874-470c-9f1b-94ff505c9b2b.png)

And why is it not using padding as before? The width is relative (just placed it in-game for demo):
![osu!_qWdqIy3USp](https://user-images.githubusercontent.com/35318437/212605345-9dc84e60-4f18-4a8e-b6d0-2f6a98661c26.gif)